src/python/WMCore/Services/WorkQueue/WorkQueue.py

```diff
@@ -191,12 +191,12 @@ def getElementsStatusAndJobsByWorkflow(self, inboxFlag=False, stale=True):
         options = {'reduce' : True, 'group_level': 2}
```

Comment:
option name is `group` either true or false (though I think group_level would work here...)